### ISSUE TITLE
bug on commit 517999

### ISSUE THREAD

**Issue by agliottone** [yourlabs/django-autocomplete-light#265]
hi,
this code:
```javascript
    this.input.parents().filter(function() {
        return $(this).css('overflow') === 'hidden';
   }).first().css('overflow', 'visible');
```
 of this changeset:
https://github.com/yourlabs/django-autocomplete-light/commit/517999a2e9ad880032679d808d72d2ece0b1f347#diff-02e96c218abe02988d4453de4dd92199

broke my form
![screenshot from 2014-03-26 11 23 22](https://f.cloud.github.com/assets/1611152/2523446/5437b968-b4d1-11e3-8d3e-c4065c5d6eb9.png)

hi have resolved with:
```css
fieldset .form-row {
    overflow: visible !important;
    min-height: 26px;
}
```
do you know if this can impact badly in other parts? in case it's acceptable could this be integrated autocomplete_light?

**Comment by jpic** [comment]
Hi ! We'd like to have a release in the next ~2 weeks, would you have time to make a PR ?

Thanks for your feedback.

**Comment by jpic** [comment]
Shouldn't that be merged into Django rather ?

**Comment by jpic** [comment]
If I remove it it still doesn't work very well.

I created a testapp in branch fix/265 and you can reproduce it in http://localhost:8000/admin/admin_autocomplete_in_row/yourmodel/add/

It's really challenging, help would be awesome !

**Comment by jpic** [comment]
Unfortunately, we're victim of a bug in Django 1.8's admin CSS apparently, I could reproduce it here: https://github.com/jpic/test_input_relation

In the test_input_relation repo, you have no autocomplete light, just a charfield and an fk in the same form-row, result: the fk is completely unuseable ....

**Comment by jpic** [comment]
Fixed in 2.2.0rc1

**Comment by jpic** [comment]
Could you [try 2.2.0rc1](http://blog.yourlabs.org/post/119860038918/django-autocomplete-light-2-2-0rc1-release-notes) and let me know if the problem still occurs ?

Thanks !
